### PR TITLE
Publish standard golang coverage report

### PR DESCRIPTION
**Pull request by u5surf**
We can resolve issue #74 with https://github.com/peaceiris/actions-gh-pages

FYI: https://goropikari.hatenablog.com/entry/go_coverage_github_pages
This site is only written in Japanese, but I guess you can read some of code snippets.
I've tried to execute the actions before I made this PR, but I cannot grasp any clue which execute the actions before making pull request.

### CODE REVIEW

**Comment by kehoecj** [comment]
Thanks for the PR @u5surf - this looks like the right solution. Based on the logs it looks like it's trying to publish to the main branch which CI does not have access to. Would it be possible to use the main branch of your fork to publish a test report over there so we can see what the published HTML would look like?

**Comment by u5surf** [comment]
@kehoecj 

>  Based on the logs it looks like it's trying to publish to the main branch

The below log says that it had pushed gh-pages branches but it cannot find the branch.
https://github.com/Boeing/config-file-validator/actions/runs/6761535151/job/18376500596?pr=87#step:6:152

`gh-pages` is the default branch to push the document which is generated by the github actions(in this case it is coverage http file).
https://github.com/peaceiris/actions-gh-pages#%EF%B8%8F-set-another-github-pages-branch-publish_branch
In short the document say that we should prepare the branch which can be pushed the document.

**Comment by kehoecj** [comment]
@u5surf I'm going to close this. I'm working to get coveralls approved to use for this project. I think that's the right long-term solution (which you had mentioned in the issue thread earlier)
